### DISCUSSION TITLE
[新艾利都快讯]

### DISCUSSION THREAD

**Discussion by Soldier11-ObsidianBarracks** [share121/inter-knot#1669] [其他]
新艾利都历0001年8月12日下午两点四十分，新艾丽都主席邀请各位参加第二届新艾丽都人民代表大会的场外会议
位置在光映广场光映警署分局的门口，我们将在0001年8月13日下午五点三十分在映广场光映警署分局同步直播代表大会现场，当然各位也可以从手机上观看直播，直播账号为光映警署

**Comment by JustOldJohn** [comment]
JM搜不到啊

**Comment by RS64671012** [comment]
> JM搜不到啊

没你事，玩儿去吧

**Comment by gaojiaobei** [comment]
好看，爱看

**Comment by ZhuYuan-148** [comment]
今日会和前辈去各街道宣传答疑，请大家积极参与！

**Comment by lllliufeng** [comment]
来了来了
